org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/format/experimental/XMLFormatterJoinCommentLinesTest.java

```diff
@@ -90,20 +90,19 @@ public void testJoinCommentLinesNested() throws BadLocationException {
 
 	@Test
 	public void testCommentFormatSameLine() throws BadLocationException {
-		String content = "<a>" + lineSeparator() + //
-				" Content" + lineSeparator() + //
+		String content = "<a>\n" + //
```

Comment:
Make a duplicate of this test with joinContentLines = true (if there isn't one already)